### ISSUE TITLE
Build status in README broken

### ISSUE THREAD

**Issue by CrashyBang** [base16-builder/base16-builder#141]
The README build status is currently not showing the linux build, as a side note I notice both windows and linux are looking for alexbooker/base16-builder as opposed base16-builder/base16-builder does this need to be updated?

Cheers.

**Comment by aloisdg** [comment]
Yes indeed. The project used to be host in alex's page. Could you provide a fix? Thank you ☺
